### ISSUE TITLE
fix competition in Rôle_monarchique

### ISSUE THREAD

**Issue by VladimirAlexiev** [dbpedia/mappings-tracker#45]
@jplu: "competition" in http://mappings.dbpedia.org/index.php/Mapping_fr:Infobox_Rôle_monarchique is clearly wrong

**Comment by jplu** [comment]
Hi @VladimirAlexiev 

Ideed, it is wrong, I corrected it, tell me if you think the new mapping is better.

**Comment by VladimirAlexiev** [comment]
@jplu: Unfortunately not because "occupation" is an objectProperty not a DataProperty.
I looked into it: #49. If we turn it to a DataProperty your mapping will be correct.
Can you help with "check randomly" in #49 ?

**Comment by VladimirAlexiev** [comment]
@jplu: I think the prop you need is "office". But check it's a data prop.

**Comment by jplu** [comment]
I don't think so, as in this case it is really a function or a position in a monarchy system. But I will check randomly in #49 the results of occupation to see if we can use it as a DataProperty.

Anyway in the infobox model of "Role Monarchique", the value of "fonction" properties are internal links.

**Comment by VladimirAlexiev** [comment]
You cannot use occupation as a DataProperty since it's declared as an ObjectProperty. If the extractor finds no links there, it'll emit nothing. ("office" is indeed a DataProperty)

"office" sounds similar enough to "function or position" to me. Here the lack of definition really hurts us :-( This prop is used in http://mappings.dbpedia.org/index.php/Mapping_en:Infobox_officeholder.
Eg https://en.wikipedia.org/w/index.php?title=Abraham_Lincoln&action=edit has:

```
|office=President of the United States
|office2=Member of the [[U.S. House of Representatives]] from [[Illinois]]'s [[Illinois' 7th congressional district|7th]] district
|office3 = [[Illinois House of Representatives|Member of the Illinois House of Representatives]]
```

Please give a FR example that's drastically different.

**Comment by jplu** [comment]
It is indeed very similar, so I replaced `occupation` by `office`.

**Comment by VladimirAlexiev** [comment]
resolve then?
